### DISCUSSION TITLE
Some uploads are collapsed

### DISCUSSION THREAD

**Discussion by thenormal** [nicotine-plus/nicotine-plus#2912] [Q&A]
Hello everyone. I have been using the latest version of Nicotine+ for a few days on a Windows 10 platform, and I noticed something that did not happen with the version I was using for many months (I think it was 3.2.9).

Anyhow, what I experience is that when someone is downloading stuff from my shared folders, in the Upload tab some of the uploads are collapsed. By collapsed I mean let's say I have three different users downloading stuff from me. At least one of them is showing up as collapsed, while the others are not, hence I can see every single file they are getting. For the one that is collapsed, I have to manually click on the arrow pointing to the right, and turn it into an arrow pointing down. That allows me to see every file they are getting.

I hope I made myself pretty clear. I tried to look under Options to see whether or not there is something I can change to disable such behavior, but couldn't locate it.

Regards.

**Comment by slook** [comment]
In the Upload tab the "Expand / Collapse All" button is located at the upper-right:

![image](https://github.com/nicotine-plus/nicotine-plus/assets/88614182/b7fd9bf4-a52d-4d4a-9509-4e124a880212)

**Reply by mathiascode** [reply]
I think the issue is that some recently added uploads remain collapsed while Expand All is enabled.

**Comment by mathiascode** [comment]
Should be fixed in the latest testing build. Can you try it and verify that everything works as you'd expect?
https://nicotine-plus.org/doc/TESTING.html#windows

Edit: nevermind, the builds didn't succeed yet. You'll have to wait a while before they're ready.

**Reply by mathiascode** [reply]
Builds are updated again.
